src/main/java/com/sparkplatform/api/SparkSession.java

```diff
@@ -99,6 +99,7 @@ public int getExpiresIn()
 	public void setExpiresIn(int expiresIn)
 	{
 		this.expiresIn = expiresIn;
+		this.refreshTimeout = expiresIn / 2;
```

Comment:
I thought of the case when **expiresIn** would be set higher than 1 day, in that case it would make sense to refresh access token later...